features/dockertools.adoc

```diff
@@ -0,0 +1,76 @@
+= Docker Tools
+:page-layout: features
+:page-product_id: jbt_core
+:page-feature_id: docker
+:page-feature_image_url: images/docker-logo.png
+:page-feature_order: 1
+:page-feature_tagline: Containerize your applications
+:page-issues_url: https://issues.jboss.org/browse/JBIDE/component/12321304
+
+== Docker Tools
+=== Connection Wizard
```

Comment:
maybe have a small introductory paragraph about what docker is and how it relate to this tooling. similar we have for arquillian, forge and livereload etc.